spec/imgflo-url.coffee

```
@@ -152,6 +156,33 @@ describe 'imgflo-url', ->
 
           expect(url).to.equal "https://imgflo.herokuapp.com/graph/#{process.env.IMGFLO_KEY}/#{process.env.IMGFLO_TOKEN_03}/passthrough.jpg?input=https%3A%2F%2Fv.cdn.vine.co%2Fr%2Fvideos%2FB5B06468B91176403722801139712_342c9a1c624.1.5.15775156368984795444.mp4.jpg%3FversionId%3DedU_LrAtIFsGvZj.Fgi0Si1bem68tBlk"
 
+      context 'with a url with uppercase JPG extension', ->
+
+        it 'should produce the correct URL', ->
+          config = getConfig()
+
+          params =
+            input: 'http://1.bp.blogspot.com/-1h8jX1lfGJc/UnlTZ3Fsq6I/AAAAAAAAAOo/dw7IXnJBO5A/s1600/IMG_1722.JPG'
+
+          url = imgflo config, 'passthrough', params
+
+          expect(url).to.contain 'passthrough.jpg'
```

Is this redundant or just to fail early?